### ISSUE TITLE
"No such file or directory - node" after upgrading to 4.0.0beta4

### ISSUE THREAD

**Issue by slhck** [rails/sprockets#490]
I am using Sprockets 4.0.0beta4 and had to install `less-rails` from [this branch](https://github.com/brendon/less-rails/tree/fix-sprockets-loading) to get Unicorn to start. What I did was `bundle install`, and now I'm running my app in the development environment.

I get the following error when opening my root page:

    Errno::ENOENT in Home#index
    No such file or directory - node

The error is in my `app/assets/config/manifest.js` file, line 2:

```
//= link_tree ../images
//= link_directory ../javascripts .js
//= link_directory ../stylesheets .css
```

My Gemfile.lock is here: https://pastebin.com/raw/vfhxMtqZ – unfortunately I cannot provide a sample application at this time (closed source). I was following [this guide](http://nandovieira.com/using-es2015-with-asset-pipeline-on-ruby-on-rails) to install the Sprockets 4 beta and the `babel-schmooze-sprockets` Gem.

Any idea how I could troubleshoot this?

**Comment by rafaelfranca** [comment]
Do you have node js installed in your machine?

**Comment by slhck** [comment]
Thanks for the hint. Actually, no, not on the Docker image that I'm using for dev work. Let me try again tomorrow.

Where in the docs does it actually say that you need node.js installed, just in case I missed that?

**Comment by rafaelfranca** [comment]
This is not a requirement of sprockets, but it is from `babel-schmooze-sprockets`.

**Comment by slhck** [comment]
I see, they don't say that either. Will ask there … thanks for the help!
